src/app/clusters/fan-control-server/fan-control-server.h

```diff
@@ -19,13 +19,23 @@
 
 #include "fan-control-delegate.h"
 #include <app-common/zap-generated/cluster-objects.h>
+#include <app/AttributeAccessInterface.h>
 #include <app/util/af-types.h>
 
 namespace chip {
 namespace app {
 namespace Clusters {
 namespace FanControl {
 
+class FanControlAttributeAccessInterface : public AttributeAccessInterface
```

Comment:
This needs documentation.  The fact that all users of this cluster now need to inherit from this thing, and will get non-spec-compliant behavior unless they do (which is an API break, by the way), is _extremely_ non-obvious.

What should have happened here is that if we need an AttributeAccessInterface registered for proper cluster functionality then the _cluster_ needs to register it.  And if we have things the app needs to handle, the cluster should be calling out into the app for those things, via the relevant delegate interface, instead of the app implementing AttributeAccessInterface directly.